Project.toml

```diff
@@ -15,21 +17,21 @@ SetRounding = "3cc68bcd-71a2-5612-b932-767ffbe40ab0"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
+Compat = "3.41, 4"
 CRlibm = "0.7, 0.8, 1"
-IntervalContractors = "0.4"
+EnumX = "1"
 FastRounding = "0.2, 0.3"
-Polynomials = "0.7, 1, 2, 3"
+Polynomials = ">= 1.0"
 RecipesBase = "1.0"
 RoundingEmulator = "0.2"
 SetRounding = "0.2"
-StaticArrays = "0.8, 0.9, 0.10, 0.11, 0.12, 1.0"
-julia = "1.5"
+StaticArrays = "1.0"
```

Comment:
I think this constrains to versions of the form 1.0.x... Current version of `StaticArrays` is v1.6.0. A similar comment applies to `Polynomials`, and `CRlibm`.

Reply:
`1.0` is the same as `1` and stands for the interval [1.0.0, 2.0.0).
[Source](https://pkgdocs.julialang.org/v1/compatibility/#Caret-specifiers); note that the caret (^) is the default.

Reply:
Thanks for the clarification 👍